.github/workflows/end-to-end-tests.yml

```diff
@@ -134,6 +134,14 @@ jobs:
             path: artifacts
             if-no-files-found: ignore
 
+      - name: Upload the build directory as GitHub artifact
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        if: always()
+        with:
+            name: wordpress-build
+            path: build
+            if-no-files-found: ignore
```

Comment:
We should probably use `error` here. If the artifact fails to upload, that's a problem.

Also, failed workflows automatically restart once. So this would force a failure and retry to be sure there's not something else going on.